src/test/java/uk/ac/cam/cl/dtg/segue/etl/ContentIndexerTest.java

```diff
@@ -102,8 +99,8 @@ public void buildSearchIndexes_sendContentToSearchProvider_checkSearchProviderIs
     Map<String, String> someUnitsMapRaw2 = ImmutableMap.of("cleanKey", "km", "unit", "km");
 
     Date someCreatedDate = new Date();
-    Map versionMeta = ImmutableMap.of("version", INITIAL_VERSION, "created", someCreatedDate.toString());
-    Map tagsMeta = ImmutableMap.of("tags", someTagsList);
+    Map<String, String> versionMeta = Map.of("version", INITIAL_VERSION, "created", someCreatedDate.toString());
```

Comment:
Nice change 👏 👏 👏. So you're removing the raw usage of Map, and you're also using Java's native `Map.of` which also returns an immutable map, right? Rather than using the Google utility. Would you mind making the same change in the rest of this file, please? It's a good one. Hopefully this can be done automatically with your IDE.

Reply:
I think that was one of those IDE suggestions actually, so it should hopefully be simple enough to apply it elsewhere...

Reply:
Hmmm... Looks like there's a few other google collection references in there. Should I have a look at replacing those with Java versions as well?

Reply:
It's up to you, but you don't need to go out of your way to do the changes. If you decide to do them, please investigate first to make sure that what you're replacing is actually equivalent. Sometimes there are subtleties and there are no one to one replacements.

Reply:
Huh. Apparently the google-api-client newHashMap/Set function literally just invoke the java-util versions?
`public static <K, V> HashMap<K, V> newHashMap() {
    return new HashMap();
  }`
`public static <E> HashSet<E> newHashSet() {
    return new HashSet();
  }`
Might just swap those out in that case...